key.core/src/main/java/de/uka/ilkd/key/logic/op/ProgramVariable.java

```diff
@@ -262,4 +274,8 @@ public MatchConditions match(SourceData source, MatchConditions matchCond) {
      */
     abstract public Operator rename(Name name);
 
+    public Object getOrigin() {
```

Comment:
I plea for a different name, as this collides `HasOrigin#getOrigin`, which uses the parser to give a (debug) information where a symbol was defined.

Reply:
I have changed it to `createdByRuleApp`.